### PR TITLE
chore: upgrade to webapi v1.6

### PR DESCRIPTION
**Pull request by stijnmoreels**
Upgrade to the newest version of Arcus WebAPI to make use of the HTTP client correlation tracking feautres.

### CODE REVIEW

File: src/Arcus.API.Market/Repositories/OrderRepository.cs [hunk 1]
```diff
@@ -60,8 +62,20 @@ public async Task OrderBaconAsync(int amount)
                 finally
                 {
                     var serviceBusEndpoint = _serviceBusOrderSender.FullyQualifiedNamespace;
+                    string entityPath = _serviceBusOrderSender.EntityPath;
                     _logger.LogInformation($"Done sending at {DateTimeOffset.UtcNow}");
-                    _logger.LogServiceBusQueueDependency(_serviceBusOrderSender.EntityPath, isSuccessful, serviceBusDependencyMeasurement, dependencyId: newDependencyId);
+                    //_logger.LogServiceBusQueueDependency(serviceBusEndpoint, entityPath, isSuccessful, serviceBusDependencyMeasurement, dependencyId: newDependencyId);
+                    _logger.LogWarning(MessageFormats.DependencyFormat, new DependencyLogEntry(
```
Comment (stijnmoreels) [inline src/Arcus.API.Market/Repositories/OrderRepository.cs:68]: Problem with the Service Bus dependency tracking: we should only add the entity path to the target name, not the namespace. Fixed in https://github.com/arcus-azure/arcus.observability/pull/445

Comment (fgheysels) [reply src/Arcus.API.Market/Repositories/OrderRepository.cs:68]: Should we then first release a new version of observability that contains that fix, before updating this POC ?

Comment (stijnmoreels) [reply src/Arcus.API.Market/Repositories/OrderRepository.cs:68]: We could, indeed. 👍

Comment (stijnmoreels) [reply src/Arcus.API.Market/Repositories/OrderRepository.cs:68]: We should already releasing v2.6 of observability as we have several new features on `master` beside this fix.

Comment (stijnmoreels) [reply src/Arcus.API.Market/Repositories/OrderRepository.cs]: Upgraded with Observability v2.6 and WebAPI v1.6.1.

Comment (stijnmoreels) [reply src/Arcus.API.Market/Repositories/OrderRepository.cs]: Still correct application map and request linking. 👍

**Review by fgheysels — APPROVED** [review]

**Pull request merged**